docs/guides/airnode/post-processing/index.md

```diff
@@ -0,0 +1,256 @@
+[2023-06-06 21:49:44.607] INFO API gateway server running on "http://localhost:3000"
+
+```
+
+## 6. Call the Airnode endpoint
+
+Use CURL and call the Airnode endpoint that gets the market data from the
+CoinGecko operation `/coins/{coin}/market_chart`. The CURL command will call the
+httpGateway exposed by the Airnode. This avoids the need to use a smart contract
+to get the response. The
+[httpGateway](/reference/airnode/latest/understand/http-gateways.md) is a
+testing tool not normally enabled on production systems.
+
+```sh
+# For Windows CMD replace line termination marker \ with ^
+curl -X POST \
+  -d '{"parameters":{"air_coin":"api3","air_vs_currency":"usd"}}' \
+  -H 'Content-Type: application/json' \
+  'http://localhost:3000/http-data/01234567-abcd-abcd-abcd-012345678abc/0x2de6e288ed16965b68a62d4b2a747b094b3c857941285e625bed3a7be31445e4' | json_pp
+
+# response
+{
+   "encodedValue" : "0x00000000000000000000000000000000000000000000000000000000001309ed00000000000000000000000000000000000000000000000000000000065b158a",
+   "rawValue" : {
+      "avg_market_cap" : "106632586.97711395",
+      "avg_price" : "1.2477250309068642",
+      "market_caps" : [
+         [
+            1685577600000,
+            105212548.064737
+         ],
+         ...
+      ],
+      "prices" : [
+         [
+            1685577600000,
+            1.23324995186942
+         ],
+         ...
+      ],
+      "total_volumes" : [
+         [
+            1685577600000,
+            3074677.52645394
+         ],
+         ...
+         ]
+      ]
+   },
+   "values" : [
+      "1247725",
+      "106632586"
+   ]
+}
+```
+
+The data retrieved from the API provider CoinGecko (`market_caps`, `prices`, and
+`total_volumes` ) was added to the `rawValue` field. Then the `rawValue` (as the
+`input` variable) was "piped" through the `postProcessingSpecifications` user
+defined code which in turn calculated the price and market cap averages which
+were added back into `rawValue` via the `output` variable.
+
+- `"avg_price" : "1.2477250309068642"`
+- `"avg_market_cap" : "106632586.97711395"`
+
+After the `postProcessingSpecifications` user defined code exited Airnode used
+the mappings of the `reservedParameters` field in the `config.json` file to
+determine what data should go into the `values` field. Airnode then encoded the
+values into the `encodedValue` field suitable for use on-chain.
+
+## 7. Verify `encodedValue`
+
+Verify that `encodedValue` actually contains the two numbers in the `values`
+field. Copy and paste the `encodedValue` string you received after executing the
+Airnode endpoint into the form below and select the **Decode** button.
+
+<EthersAbiCoder types="int256,int256"/>
+
+## Summary
+
+In this guide you extended the data from the API operation with additional
+summary values. For more examples on how to use `preProcessingSpecifications`
```

Comment:
Suggestion: "summary values using post processing. For more..."

Reply:
OK